app/views/candidate_mailer/reinstated_offer.text.erb

```diff
@@ -8,16 +8,10 @@ The course starts in <%= @course_option.course.start_date.to_fs(:month_and_year)
 
   <%= render "candidate_mailer/offer_conditions" %>
 
-<% else %>
-  They’ll let you know if they need further information before you start training.
-<% end %>
+  Sign into your account to check the progress of your offer conditions.
 
-<% if @application_form.show_new_reference_flow? %>
-
-  <%= @provider_name %> also needs to check your references and DBS.
-
-  [Sign in](<%= candidate_magic_link(@application_choice.application_form.candidate) %>) to check the progress of your reference requests and offer conditions.
+  [Sign into your account](<%= candidate_magic_link(@application_choice.application_form.candidate) %>).
 
 <% else %>
```

Comment:
Think this `else` and the following `end` need to go, as you’ve removed the `if` statement?

Reply:
Oh actually I misread this - the `if` has gone, but so has the previous `else` and `end`, so this might be ok?

Reply:
I think it's OK, as long as we're fine to remove the logic giving different content for the old and new reference processes.

Reply:
@JR-G this might need your eyes over it too if poss.

Reply:
Sounds fine to me, as we're past the point of needing the old references flow 👍